### PR TITLE
Direction page made

### LINKED ISSUE #10: Directions section
- Make the title text and the empty div
- work on filer under src/components/home/Directions.jsx
Screenshot 2024-06-28 at 11 23 57 AM

### PR DESCRIPTION
**Pull request by Sangha0822**
How it looks just the page:

스크린샷 2024-07-04 오후 2 17 39

How it looks with other navigation pages:
스크린샷 2024-07-04 오후 2 17 54

Closes #10

### CODE REVIEW

**Comment by Sangha0822** [comment]
I did not work on responsiveness on this page yet

**Comment by Sangha0822** [comment]
May need update on fonts for the text as well.

File: src/components/home/Directions.jsx [hunk 1]
```diff
@@ -1,5 +1,16 @@
+import React from "react";
+
 const Directions = () => {
-  return <div>Directions</div>;
+  return (
+    <div className=" flex flex-col justify-center items-center bg-[#83694D] h-screen">
+      <div className="text-white font-extrabold text-6xl ">
+        How to get to Our Garden
+      </div>
+      <div className=" h-[70%] w-[90%] bg-[#C0BDA5] mt-[3%] mb-[3%] border-2 border-[#3E2913] rounded-lg">
```
Comment (fardinzam) [inline src/components/home/Directions.jsx:9]: don't do a top and bottom margin separately, do it at the same time with my-[3%]

File: src/components/home/Directions.jsx [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+        {/* Div for rectangular empty div space*/}
```
Comment (fardinzam) [inline src/components/home/Directions.jsx:10]: remove comments

**Pull request merged**